docs/plain-bundle-spec.md

```diff
@@ -0,0 +1,138 @@
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for a [Bundle](/README.md#Bundle)
+resource.
+
+A plain bundle is simply a collection of static, arbitrary, Kubernetes YAML manifests in a given directory. A plain
+bundle can consist of a container image, a directory in a git repository, or any other content source that
+the [plain bundle provisioner](/internal/provisioner/plain/README.md) supports.
+
+The currently implemented plain bundle format is the `plain+v0` format. The name of the bundle format, `plain+v0`
+combines the type of bundle (plain) with the current schema version (v0).
+The [plain bundle provisioner](/internal/provisioner/plain/README.md) is able to source `plain+v0` bundles and install
+them onto a Kubernetes cluster.
+
+> Note: the plain+v0 bundle format is at schema version v0, which means it's an experimental format that is subject
+> to change.
+
+Supported source types for a plain bundle currently include the following:
+
+* A container image
+* A directory in a git repository
+
+Additional source types, such as a local volume or a generic URI-based resource, are on the roadmap. These source types
+all present the same content, a directory containing static Kubernetes YAML manifests, in a different ways.
+
+## Example
+
+For example, below is a minimal example of a Dockerfile that builds a `plain+v0` image-based bundle from a directory
+containing static Kubernetes manifests.
+
+```dockerfile
+FROM scratch
+COPY /manifests /manifests
+```
+
+where the given `manifests` directory contains the Kubernetes resources required to deploy an application, for example:
+
+```
+manifests
+├── namespace.yaml
+├── cluster_role.yaml
+├── role.yaml
+├── serviceaccount.yaml
+├── cluster_role_binding.yaml
+├── role_binding.yaml
+└── deployment.yaml
+```
+
+For an example of a directory backing a git-based bundle,
+see [the combo manifests directory](https://github.com/operator-framework/combo/tree/main/manifests). Any directory that
+contains only static Kubernetes manifests checked into a git repository accessible via a remote URL can be considered a
+plain bundle and sourced by the plain provisioner.
+
+> Note: there must be at least one resource in the manifests directory in order for the bundle to be a valid
+> plain+v0 bundle.
+
+## Technical Details
+
+* The static manifests must be located in the root-level /manifests directory in a bundle image for the bundle to be a
+  valid `plain+v0` bundle that the provisioner can unpack. A plain bundle image without a /manifests directory is
+  invalid and will not be successfully unpacked onto the cluster. For a git-based plain bundle, this limitation does not
+  exist, and the manifests can be in any directory in the repository. The manifest directory is assumed to be
+  ./manifests in a git-based plain bundle but can be overwritten.
+* The /manifests directory may contain additional sub-directories which contain static manifests.
```

Comment:
I'd suggest that we start with a flat directory for now. We can always allow subdirectories in the future without making a breaking change, but we can't go the other way around. Supporting subdirectories might complicate the ordering conversation we were having earlier.

Reply:
+1

Reply:
My suggestion for combo for now would be:

```
combo
└── bundle
    └── manifests
        └── manifest.yaml
```

in case we put anything else in the bundle spec that would be a sibling of `manifests`. If we collapse it down and remove the `bundle` directory from the tree, then there's potential for bundle metadata stuff to be intermingled with other operator source code.

That's kind of why I like the idea of `bundle.yaml` with that name; it _could_ live at that root level and not be some non-descript file sitting there.